static/app/components/replays/replaysFeatureBadge.tsx

```diff
@@ -0,0 +1,7 @@
+import FeatureBadge from 'sentry/components/featureBadge';
+
+function ReplaysFeatureBadge() {
+  return <FeatureBadge type="alpha" noTooltip />;
```

Comment:
Can you follow-up in a separate PR to update other uses of this badge?

![image](https://user-images.githubusercontent.com/79684/181606685-560a8a4e-f5aa-47d2-802b-ea373eca07b7.png)

Reply:
https://github.com/getsentry/sentry/issues/37021
https://github.com/getsentry/sentry/issues/37144